attributes/default.rb

```diff
@@ -1,25 +1,28 @@
 if platform?('windows')
-  default['nexus3']['url'] = 'http://download.sonatype.com/nexus/3/latest-win64.zip'
-  default['nexus3']['path'] = ENV['SYSTEMDRIVE']
+  default['nexus3']['path'] = 'C:\Nexus3'
+  default['nexus3']['group'] = 'Administrators'
 else
-  default['nexus3']['url'] = 'http://download.sonatype.com/nexus/3/latest-unix.tar.gz'
   default['nexus3']['path'] = '/opt'
+  default['nexus3']['group'] = 'nexus'
 end
+# Download URL is defined in the resource but you can override it with the default['nexus3']['url'] attribute
+default['nexus3']['url'] = nil
```

Comment:
Fixed. Do you want to also include the matching sha256sum or should I leave nil as optional?